backend/amd64/emit.mlp

```diff
@@ -722,7 +722,9 @@ let emit_instr fallthrough i =
   | Lop(Itailcall_imm { func; label_after; }) ->
       begin
         if func = !function_name then
-          I.jmp (label !tailrec_entry_point)
+          match !tailrec_entry_point with
```

Comment:
Can you please make this change to other `emit.mlp` files in "backend" subdirectory? It'd be good to keep them in sync even if it's hard to test them at the moment.

Reply:
Indeed, I thought the CI would build the `check_all_arches` target,
or its dune equivalent.